### DISCUSSION TITLE
REQ: Add hud elements for kills and secrets

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#524] [Ideas]
See https://github.com/ezQuake/ezquake-source/issues/206

**Comment by emilyilzrih761-ops** [comment]
Spirit Casino ist ein modernes Online‑Casino, das Spielern in Deutschland und international ein spannendes Glücksspiel‑Erlebnis mit Echtgeld bietet. Die Plattform überzeugt durch eine große Auswahl an Slots, Live‑Dealer‑Spielen und Tischspielen, vielfältige Zahlungsmethoden inklusive Kryptowährungen sowie attraktive Bonusangebote wie https://spiritcasinos.de/ großzügige Willkommenspakete und Freispiele. Für Komfort sorgt ein 24/7‑Support und ein responsives Design für mobiles Spielen. Spieler schätzen die benutzerfreundliche Oberfläche, schnelle Auszahlungen und regelmäßige Promotionen, die das Spielen abwechslungsreich und unterhaltsam machen.

**Comment by AnthonyPatterson512** [comment]
Um HUD-Elemente für Kills und Secrets hinzuzufügen, kannst du einfach zwei Zähler im Spielstatus führen, die bei bestimmten Ereignissen aktualisiert werden. Immer wenn ein Gegner besiegt wird, erhöhst du den Kill-Counter, und bei gefundenen Geheimnissen den entsprechenden Wert. Diese Zahlen lassen sich dann als Text im Bildschirmrand anzeigen, etwa [cresus-casinos.ch](https://cresus-casinos.ch/) oben rechts oder links. Wichtig ist, dass die Anzeige regelmäßig aktualisiert wird und klar lesbar bleibt. Trenne dabei Logik und Darstellung, damit du das HUD später leicht anpassen kannst.
